### ISSUE TITLE
Toggle node via keyboard

### ISSUE THREAD

**Issue by ckosmowski** [vakata/jstree#559]
If i navigate the tree via the keyboard (tab or error keys) and then toggle a node with the spacebar to open it and display its children it only works one time i cannot toggle it multiple times and it seems the keyboard focus gets lost from the tree.

So to reproduce:
- Navigate to a node containing children with the keyboard
- open it with the spacebar
- close it with the spacebar
- open it again with the spacebar
- no keyboard actions are possible anymore on the tree

I tested this in the current Chrome and IE11
